### PR TITLE
Mixtrack Pro 3 controller - Autoloop adjustments

### PR DESCRIPTION
**Pull request by djsteph**
Adjustments to mapping of Autoloop pads to change default values from "1/8, 1/4, 1/2 and 1" beat loop to "1,2,4 and 8" beat loops; same as Serato and VDJ 

I changed 2 files by uploading them directly to Github...Sorry, this will make comparing changes harder. Changes have been tested by me with my MTP3 Pro controller

Steph

### CODE REVIEW

**Comment by sblaisot** [comment]
Please update windows uninstaller to also remove these 2 new files:
http://mixxx.org/wiki/doku.php/contributing_mappings#windows_installer_update

**Comment by sblaisot** [comment]
Oh, sorry, these are not new files, these are files that are not in the right directory. you uploaded them at the root of the project. see https://github.com/djsteph/mixxx/tree/Mixtrack-Pro-3-v1.4
you should update them in the res/controllers subdirectory

**Comment by sblaisot** [comment]
you just have to `git mv -f Numark* src/controllers` then `git commit -m "commit message"` and then `git push` to move them in the right place

File: res/controllers/Numark-Mixtrack-3.midi.xml [hunk 1]
```diff
@@ -1,919 +1,923 @@
...
-</MixxxControllerPreset>
+<?xml version='1.0' encoding='utf-8'?>
+<!-- Numark Mixtrack (Pro) 3 MIDI preset v1.4 -->
+<MixxxControllerPreset mixxxVersion="1.12.0+" schemaVersion="1">
+    <info>
+	<name>Numark Mixtrack (Pro) 3</name>
+	<author>Stéphane Morin</author>
+	<description>The Numark Mixtrack 3 and Numark Mixtrack Pro 3 are the same controller except that the Pro version has an integrated sound card.</description>
+        <forums>http://mixxx.org/forums/viewtopic.php?f=7&amp;t=7286</forums>
+<<<<<<< HEAD
```
Comment (sblaisot) [inline res/controllers/Numark-Mixtrack-3.midi.xml]: you left merge conflicts in this file

**Comment by djsteph** [comment]
I removed the conflict comments. Should be good now.

**Pull request closed without merging**